fitlins/interfaces/bids.py

```diff
@@ -142,6 +142,8 @@ class LoadBIDSModelInputSpec(BaseInterfaceInputSpec):
                          desc='BIDS dataset root directory')
     derivatives = traits.Either(traits.Bool, InputMultiPath(Directory(exists=True)),
                                 desc='Derivative folders')
+    database_path = Directory(exists=False,
+                              desc='Optional path to bids database directory.')
```

Comment:
Here, you're setting this up as an alternative to bids_dir/derivatives, but in `_run_interface`, you're replacing them entirely. We should be consistent about which one we're doing.

Reply:
Would you prefer it to be optional or to have `LoadBIDSModel` and `BIDSSelect` only work if passed a `database_path`?

Reply:
Optional at the CLI level for sure. But we could set up a default internal path (don't we already do that?).